content/apps/limits.md

```diff
@@ -0,0 +1,71 @@
+---
+menu:
+  main:
+    parent: apps
+title: Application Limits
+weight: -10
+---
+
+
+As a Cloud Foundry user you’re free to divide your [organization's quota]({{< relref "intro/pricing/quotas.md" >}}) capacity amongst your applications as you see fit by way of application limits. Application limits are specified in your application manifest or as `cf push` command line options.
+
+## Application Limit Options:
```

Comment:
Minor: mind using sentence case for the headings and removing the trailing colons?

https://pages.18f.gov/content-guide/capitalization/#headings